packages/react-interactive-tab/package.json

```diff
@@ -2,13 +2,15 @@
   "name": "@fluentui-contrib/react-interactive-tab",
   "version": "0.0.1",
   "dependencies": {
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
+    "@fluentui/react-tabs": ">=9.5.0 < 10.0.0",
```

Comment:
see reasoning in previous comment

```suggestion
```

Reply:
There will be an eslint error regarding the imported package is not in package.json https://github.com/microsoft/fluentui-contrib/pull/217/files#diff-e57b00fdd8526b00b7a61fa87c9a06e82c8a87049bf47872dc533491ed54b2b6R3

Reply:
yeah, in this case that's expected and we can ignore (for benefit of users) - that rule is naive in this sense as it doesn't understand dependencies